HtmlMesh/src/html-mesh.ts

```diff
@@ -6,6 +6,114 @@ import { PointerEventsCaptureBehavior } from "./pointer-events-capture-behavior"
 import { Scene } from "@babylonjs/core";
 import { Logger } from "@babylonjs/core/Misc/logger";
 
+export enum SizeFit {
+    none = 1,
+    contain = 2,
+    cover = 3,
+    stretch = 4
+}
+
+export type SizeFitHandler = {
```

Comment:
How about FitStrategy with functions wrapElement and updateSize.  Let's make createElement without side effects so it accepts the element and returns the sizing wrapper.   updateSize would modify the passed in element.